homeassistant/components/zimi/coordinator.py

```diff
@@ -0,0 +1,114 @@
+"""Zimi Controller coordinator class device."""
+
+import logging
+import pprint
+
+from zcc import ControlPoint, ControlPointDescription, ControlPointError
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_PORT
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+from .const import PLATFORMS
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_connect_to_coordinator(
```

Comment:
```suggestion
async def async_connect_to_controller(
```
to clarify it's not the HA coordinator

Reply:
I'm also thinking move that somewhere else `helpers.py` maybe?